WordPressKit/Activity.swift

```diff
@@ -158,6 +158,30 @@ public struct ActivityStatus {
     public static let warning = "warning"
 }
 
+public class ActivityGroup {
+    public let name: String
```

Comment:
We also need to store the key here. Eg.: for "Posts and Pages" the key is "post", which is the one we use to actually filters results. Eg.:

`https://public-api.wordpress.com/wpcom/v2/sites/165243437/activity?_envelope=1&aggregate=true&on=2020-11-17%2003%3A00%3A00&group%5B%5D=post&number=1000`

Reply:
Thanks for the hint 😄 
Fixed! 5b7f2ed